Sources/Sentry/SentryFileManager.m

```diff
@@ -81,6 +82,9 @@ - (nullable instancetype)initWithOptions:(SentryOptions *)options
 
         self.currentFileCounter = 0;
         self.maxEnvelopes = options.maxCacheItems;
+
+        dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_LOW, 0),
```

Comment:
`m`: I think we could delay this for a couple of seconds when the app and the SDK finished launching. What about 10 seconds? You then could use the `DispatchQueueWrapper` to improve testability, so you don't have to wait for 10 seconds.

Reply:
Is that better than just using `DISPATCH_QUEUE_PRIORITY_LOW`?

Reply:
Even with `DISPATCH_QUEUE_PRIORITY_LOW` it adds overhead at the beginning of the app start. This has a really low priority and can run later.

Reply:
That is true, but then again after 10 seconds you might be starting some heavy operation as well 😅

In fact it's probably more likely that you're using the app and navigation between different screens 10 seconds after starting the app vs immediately after starting the app. Not sure about this!

Reply:
I've used SentryDispatchQueueWrapper, but I am not sure the 10 second delay is a great idea. Also, SentryDispatchQueueWrapper uses a default queue instead of a low priority queue.